src/helpers/batfish/split-pages.js

```diff
@@ -53,7 +53,7 @@ function combine(pages, field) {
 }
 
 function getHeadings(page) {
-  let headings = [];
+  const headings = [];
```

Comment:
Wow. You don't have to use `let` if you're going to push to an array? I never knew...